addons/mail/static/src/new/activity/activity.js

```diff
@@ -17,18 +17,21 @@ import { _t } from "@web/core/l10n/translation";
  * @typedef {Object} Props
  * @property {import("./activity_model").Activity} data
  * @property {function} [onUpdate]
+ * @property {function} reloadParentView
  * @extends {Component<Props, Env>}
  */
 export class Activity extends Component {
     static components = { ActivityMailTemplate, FileUploader };
-    static props = ["data", "onUpdate?"];
+    static props = ["data", "onUpdate?", "reloadParentView"];
```

Comment:
I think `reloadParentView` could just be renamed to `reloadView`. It would be easier to read.